packages/vuetify/build/constants.js

```diff
@@ -0,0 +1,17 @@
+import packageJson from '../package.json' with { type: 'json' }
```

Comment:
Is this going to cause warnings like assert used to?

Reply:
It shouldn't it is just used at build script.

I forgot to mention I move this to the new contants.js module to be reused in unimport and unplugin vue components rollup configuration file.

Reply:
`assert` was renamed to `with` in node 18.20